packages/gatsby-theme-instore/src/typings/index.d.ts

```diff
@@ -0,0 +1,95 @@
+declare type Maybe<T> = T | undefined | null
```

Comment:
Why this file?

Reply:
I don't know, this file came when I run `yarn upgrade`